src/appleseed/renderer/modeling/project/projectfilewriter.cpp

```diff
@@ -110,7 +110,6 @@ namespace renderer
 namespace
 {
     // Floating-point formatting settings.
-    const char* VectorFormat     = "%.15f";
```

Comment:
You're right, we don't. The vectors I had in mind were rotation axes for `<rotation>` elements and scaling factors for `<scaling>` elements, but these are collapsed to matrices when writing project files.